controller-server/src/test/java/com/yahoo/vespa/hosted/controller/deployment/DeploymentTester.java

```diff
@@ -90,7 +87,7 @@ public DeploymentTester(ControllerTester controllerTester) {
         cloud = (MockTesterCloud) tester.controller().jobController().cloud();
         var jobControl = new JobControl(tester.controller().curator());
         runner = new JobRunner(tester.controller(), Duration.ofDays(1), jobControl,
-                               JobRunnerTest.inThreadExecutor(), new InternalStepRunner(tester.controller()));
+                               JobRunnerTest.inThreadExecutor(), new InternalStepRunner(tester.controller()), tester.clock());
```

Comment:
And the tester's clock is the same as the controller's, in this case.

Reply:
Fixed